dashboard/controller/users.js

```diff
@@ -146,7 +147,9 @@ exports.editUser = function(req, res) {
 			req.body.name = req.body.name.trim();
 			req.body.password = req.body.password.trim();
 			req.body.color = JSON.parse(req.body.color);
+			req.assert('name', common.l10n.get('UsernameInvalid')).matches(/^[a-z0-9 ]+$/i);
```

Comment:
Would be better to put it in function/variable instead of repeating the regular expression.